sample/src/main/java/com/canhub/cropper/sample/camera/presenter/SCameraPresenter.kt

```diff
@@ -92,41 +86,30 @@ internal class SCameraPresenter : SCameraContract.Presenter {
         view?.showErrorMessage("onCancel")
     }
 
-    override fun onActivityResult(resultCode: Int, requestCode: Int, data: Intent?) {
-        if (resultCode == RESULT_OK) {
-            when (requestCode) {
-                CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE -> {
-                    val bitmap = context?.let { CropImage.getActivityResult(data)?.getBitmap(it) }
-                    Log.v(
-                        "File Path",
-                        context
-                            ?.let { CropImage.getActivityResult(data)?.getUriFilePath(it) }
-                            .toString()
-                    )
```

Comment:
I know this two don't have direct value for the code, but we need to keep the logic for method testing and user example of how getting the `bitmap` and `UriFilePath` (big issues on this latest one) 

So we would need:
```
val bitmap = result.bitmap
Log.v("File Path", context?.let { result.getUriFilePath(it) }.toString())
```